docs/manual/getting_started.md

```diff
@@ -74,18 +103,21 @@ spec:
             - name: FDB_CLUSTER_FILE
               value: /mnt/config-volume/cluster-file
             volumeMounts:
-            - name: config-volume
+            - name: shared-volume
               mountPath: /mnt/config-volume
```

Comment:
doesn't match the name above?

Reply:
The init container mounts the `config-volume` which is a `ConfigMap` and copies the cluster file into the `shared-volume` which is an `EmptyDir`. The reason for this is to make the cluster file writable by the client.

Reply:
yep that makes sense. It might be helpful to have a comment for those of us who always mix these things up

Reply:
I'll add a comment in another PR, I still have to create a note on why the cluster file should always be copied and not directly be used by the application: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1958#issuecomment-1985554885.

Reply:
even better!